### PR TITLE
Added Initial doc for PyCon India 2021

### PR DESCRIPTION
**Pull request by raukadah**
It got copied from PyCon India 2020 README with some
modifications.

Signed-off-by: Chandan Kumar (raukadah) <raukdah@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,190 @@
-# inpycon2021-tasks
\ No newline at end of file
+# PyCon India 2021
+
+
+## Event Dates:
```
Comment (sayanchowdhury) [inline README.md:4]: Add the comment that this is tentative.

File: README.md [hunk 2]
```diff
@@ -1 +1,190 @@
...
+- Conference: 16th and 17th Sep, 2021
+- Workshop: 18th Sep, 2021
+- DevSprint: 18th and 19th Sep, 2021
```
Comment (sayanchowdhury) [inline README.md:7]: There will be no devsprints, right?

File: README.md [hunk 3]
```diff
@@ -1 +1,190 @@
...
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
+    * Diversity
```
Comment (sayanchowdhury) [inline README.md:14]: We talked about Diversity to be in sessions, and moving a couple of talks and if possible Panel Discussion to tracks.

Comment (sayanchowdhury) [inline README.md:14]: Also it would be better to explain the terminology:
- What is Tracks?
- What is Sessions?

File: README.md [hunk 4]
```diff
@@ -1 +1,190 @@
...
+    * Extended
```
Comment (sayanchowdhury) [inline README.md:15]: Maybe name it `.extend()` instead of Extended ?

File: README.md [hunk 5]
```diff
@@ -1 +1,190 @@
...
+- Multiple Sessions
+- Community Booths
+- Bofs
```
Comment (sayanchowdhury) [inline README.md:18]: s/Bofs/BoFs

File: README.md [hunk 6]
```diff
@@ -1 +1,190 @@
...
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
+    * Diversity
+    * Extended
+- Multiple Sessions
+- Community Booths
```
Comment (sayanchowdhury) [inline README.md:17]: Mention Community Booths in Expo Area

File: README.md [hunk 7]
```diff
@@ -1 +1,190 @@
...
+- Bofs
+
+## Working Groups
+
+Conference Chair: Chandan Kumar
+Conference Co-Chairs: Kumar Anirudha, Ritesh Agarwal
+Emritus Chair: Sayan Chowdhury
```
Comment (sayanchowdhury) [inline README.md]: s/Emritus/Emeritus

Also, add a comment that previous chairs would be reached out (if we planning to add more, it's okay to limit to one or two). Also, we need to create a doc on the roles. This is for all the tasks roles.

File: README.md [hunk 8]
```diff
@@ -1 +1,196 @@
...
+### CFP (Call For Proposals)/Keynotes
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fcfp)
+- Meeting:
+- Chair: Sukanya
+- Co-chair:
+- Shadow:
+- Members
+
+### Content and Design
```
Comment (sayanchowdhury) [inline README.md:59]: IMO, Design should be clubbed with Website under Technology

File: README.md [hunk 9]
```diff
@@ -1 +1,196 @@
...
+- [Github Issues](https://github.com/pythonindia/inpycon-blog/issues/)
+- Meeting:
+- Chair: Vipul Gupta
+- Co-chair:
+- Shadow:
+- Members
+  * Nikhil Maan
+  * Arvind
+
+### Dev Sprint/Workshops
```
Comment (sayanchowdhury) [inline README.md:70]: DevSprint is tentative this year.

File: README.md [hunk 10]
```diff
@@ -1 +1,196 @@
...
+### Diversity
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdiversity)
+- Meeting:
+- Chair: Sukanya
+- Co-chair:
+- Shadow:
+- Members
+
+### Tickets
```
Comment (sayanchowdhury) [inline README.md:151]: Since, this year FInance team will be mostly be looking into Tickets. Thoughts of clubbing them?

File: README.md [hunk 11]
```diff
@@ -1 +1,196 @@
...
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Freview)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+
+### Bofs
```
Comment (sayanchowdhury) [inline README.md:106]: BoFs

File: README.md [hunk 12]
```diff
@@ -1 +1,196 @@
...
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+## Timeline
+### TLDR;
+* PyCon India 2021 formal proposal going live on 19th Feb, 2021.
+* Website going live on 01st Mar, 2021.
+* Ticket sales going live on 01st May, 2021.
```
Comment (sayanchowdhury) [inline README.md:184]: Try this mid April?

File: README.md [hunk 13]
```diff
@@ -1 +1,196 @@
...
+* CFP opens on March 01st and Closes on last week of May, 2021.
```
Comment (sayanchowdhury) [inline README.md:185]: Also, try this mid-May. Extend if needed. Better to have a calendar gives a more better idea. We can setup a Google public calendar like we did last year.

**Comment by abhishekmishragithub** [comment]
- Add roles and responsibility (add a link to another .md file, just a suggestion)
- IMO - let 1 person be a chair/lead for a single WG and let them volunteer or be a shadow lead of another WG which interests them. (this mitigates the dependency on a single person)

**Review by abhishekmishragithub — COMMENTED** [review]
- Add roles and responsibility (add a link to another .md file, just a suggestion)
- IMO - let 1 person be a chair/lead for a single WG and let them volunteer or be a shadow lead of another WG which interests them. (this mitigates the dependency on a single person)

File: README.md (hunk 2 above)
Comment (raukadah) [reply README.md:7]: There will be devsprints, But we need to decide the format.

**Review by raukadah — COMMENTED** [review]
Thanks for reviewing, I have addressed the comments.

File: README.md [hunk 14]
```diff
@@ -1 +1,195 @@
...
+
+- Conference: 16th and 17th Sep, 2021
+- Workshop: 18th Sep, 2021
+- DevSprint: 18th and 19th Sep, 2021
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
```
Comment (sayanchowdhury) [inline README.md:14]: Why not just mention 4 general tracks. That would give more freedom to exercise and change the 4th later?

File: README.md [hunk 15]
```diff
@@ -1 +1,195 @@
...
+    * Panel Discussions
+    * .extend()
+- Sessions (Diversity)
+- Community Booths
+- BoFs
+
+## Keywords
```
Comment (sayanchowdhury) [inline README.md:21]: This should go before the Conference Structure.

File: README.md (hunk 14 above)
Comment (raukadah) [reply README.md]: Yes, make sense.

**Review by anistark — APPROVED** [review]

**Comment by palnabarun** [comment]
Merging it for now. We can iterate on changes later.

**Pull request merged**